src/components/CustomStatusBar/CustomStatusBar.constants.ts

```diff
@@ -0,0 +1,6 @@
+type StatusBarTextColorType = 'light-content' | 'dark-content';
+
+export const STATUS_BAR_TEXT_COLOR: { [key: string]: StatusBarTextColorType } = {
+	light: 'light-content',
+	dark: 'dark-content',
+};
```

Comment:
IMHO, redundant and over-complicated code

Reply:
!